tslearn/neighbors.py

```diff
@@ -115,6 +115,8 @@ class KNeighborsTimeSeries(KNeighborsTimeSeriesMixin, NearestNeighbors):
         <https://docs.scipy.org/doc/scipy/reference/spatial.distance.html>`_.
     metric_params : dict or None (default: None)
         Dictionnary of metric parameters.
+        For metrics that accept parallelization of the cross-distance matrix
+        computations, `n_jobs` is a valid `metric_params` field.
```

Comment:
Also here, it could be nice to have n_jobs as a direct parameter, and not encapsulate it within metric_params